subsys/bluetooth/audio/vocs_client.c

```diff
@@ -300,9 +300,6 @@ static uint8_t vcs_client_read_output_desc_cb(struct bt_conn *conn, uint8_t err,
 		memcpy(desc, data, length);
 		desc[length] = '\0';
 		BT_DBG("Output description: %s", log_strdup(desc));
-	} else {
-		BT_DBG("Invalid output");
-		cb_err = BT_ATT_ERR_UNLIKELY;
```

Comment:
You could remove cb_err now, since you only assign it and pass it forward.

Reply:
Re-added the cb_err as we need to check for bad reads as well.